### DISCUSSION TITLE
v1.23.3

### DISCUSSION THREAD

**Discussion by adriankumpf** [teslamate-org/teslamate#1667] [General]
### Bug Fixes

- Fix API tokens form

<hr /><em>This discussion was created from the release <a href='https://github.com/adriankumpf/teslamate/releases/tag/v1.23.3'>v1.23.3</a>.</em>

**Comment by ryawang** [comment]
Hi there,

I'm a new fan of this great project but without skills on Linux, I installed 1.22.0 on an Ubuntu system and works well so far, but while I'm trying to upgrade it to this new version, I was hit by below error messages, can you help to advise how should I do next for the fix? 
Many thanks if I can hear something cool from you.

![error](https://user-images.githubusercontent.com/85295539/120651384-4cb33300-c4b1-11eb-9f43-793d9ef59cac.JPG)

**Reply by DrMichael** [reply]
There appears to run another version of teslamate. Started from a different directory perhaps?

(BTW, and having it in /usr/local/bin is not unix like... Make a directory under your home named teslamate, put the docker-compose.yml in it and start it there. But first you have to get rid of the other instance.)

**Reply by ryawang** [reply]
Thanks DrMichael, and that probably make sense, I recalled that I was made some mistake to input the .yml file to a wrong place and start it there... what a crash I made. :(
For the action: get rid of the other instance, can you advise more about how to process? use docker-compose stop command?

Thanks.

**Reply by DrMichael** [reply]
Go to the other directory with the docker-compose.ym and do `docker-copose down`.

**Reply by ryawang** [reply]
Done, but seems it still working on :3000 webpage, need to deep dive more. Anyway, appreciate for your help!

**Comment by refrain** [comment]
Is it just me or does the trip view, when you mouse over the driving part of the state bar, no longer plot the position of the car on the map?  You used to be able to trace the entire course of the car from start to finish by sliding the mouse across the entire drive bar.  Anyone else notice this is missing now?

FYI using
teslamate 1.23.3 (but i think i noticed this back in 1.23.0)
grafana 7.5.4
